homeworks/sargis_vardanyan/homework7/DigitsToWords.java

```diff
@@ -0,0 +1,35 @@
+package homework7;
+
+
+public class DigitsToWords {
+
+	public static void main(String[] args) {
+		
+		digitsToWords(Integer.parseInt(args[0]));
+		System.out.println();
+		
+	}
+	
+	static void printArray(String[] array) {
+		
+		for(int i = 0; i < array.length; i++) {
+			System.out.print(array[i] + " ");
+		}
+		System.out.println();
+	}
+	
+	
+	static void digitsToWords(int number) {
```

Comment:
you solved different problem, please reread the problem statement in the slack channel